bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,401 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="awattar"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="awattar-bridge">
+		<label>aWATTar Bridge</label>
+		<description>Provides price data from the aWATTar API.</description>
+		<config-description uri="bridge-type:awattar:awattar-bridge">
+			<parameter name="country" type="text">
+				<label>Country</label>
+				<default>DE</default>
+				<description>Country to get prices for. Only DE (Germany) and AT (Austria) are supported.</description>
+				<options>
+					<option value="DE">DE</option>
+					<option value="AT">AT</option>
+				</options>
+			</parameter>
+
+			<parameter name="vatPercent" type="decimal">
+				<label>VAT Percent</label>
+				<description>Specifies the value added tax percentage</description>
+				<default>19</default>
+			</parameter>
+			<parameter name="basePrice" type="decimal">
+				<label>Base price</label>
+				<description>Specifies the net base price per kWh</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="timeZone" type="text">
+				<label>Time zone</label>
+				<description>Time zone to apply to the hour definitions. Default CET aligns to the aWATTar API</description>
+				<default>CET</default>
```

Comment:
Did you test if this also works if daylight saving time is active/not active?

Reply:
It works i both Winter Time and Summer Time